src_tests/test_lossy_boson_sampling_simulator.py

```diff
@@ -43,11 +44,21 @@ def test_approximate_and_exact_distribution_distance(self) -> None:
         exact_distribution_calculator = \
             BosonSamplingWithFixedLossesExactDistributionCalculator(self.experiment_configuration)
         exact_distribution = exact_distribution_calculator.calculate_exact_distribution()
-        # TR TODO: Delta in assertion should depend on samples number!
-        approximate_distribution = self.__calculate_approximate_distribution()
+
+        number_of_outcomes = len(exact_distribution_calculator.get_outcomes_in_proper_order())
+        number_of_samples = 1000
```

Comment:
Have you considered extracting these constants to some configuration, e.g. to variables at the top of the file?